benchmark/python/sparse_op.py

```diff
@@ -0,0 +1,204 @@
+import ctypes
+
+from mxnet.test_utils import *
+import scipy.sparse as sp
+import os
+import time
+import argparse
+
+from mxnet.base import check_call, _LIB
+
+parser = argparse.ArgumentParser(description="Benchmark sparse operators",
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('--num-omp-threads', type=int, default=1, help='number of omp threads to set in MXNet')
+args = parser.parse_args()
+
+
+def test_dot_real(batch_size):
+    def get_iter(path, data_shape, batch_size):
+        data_train = mx.io.LibSVMIter(data_libsvm=path,
+                                      data_shape=data_shape,
+                                      batch_size=batch_size)
+        data_iter = iter(data_train)
+        return data_iter
+
+    # dataset info
+    data_dir = os.path.join(os.getcwd(), 'data')
+    data_name = 'kdda.t.mini'
+    feature_dim = 20216830
+    density = 0.000186630098465
+    path = os.path.join(data_dir, data_name)
+
+    if not os.path.exists(os.path.join(path)):
+        raise Exception("Data is not prepared!")
+
+    size = float(os.path.getsize(path)) / (2**20)
+
+    # model
+    data_shape = (feature_dim, )
+    train_iter = get_iter(path, data_shape, batch_size)
+
+    k = 200 
```

Comment:
please rename to m instead. this should be the same as the one in synthetic benchmark. otherwise it's confusing